assets/css/main.css

```diff
@@ -80,6 +80,11 @@
   margin-top: 100px
 }
 
+.people-img {
```

Comment:
This is a known issue that I will be working on in an upcoming pull request.  The intention of this pull request is to address the sponsor's page, but because I had to splice things together to repair the merge issue, this might have slipped in there.  I'm going to leave this alone for now.  Look for fixes in next pull request.